### PR TITLE
NAS-103188

### PR DESCRIPTION
**Pull request by dkmullen**
Remove some AD fields, add cert validation checkbox to AD, LDAP

Definitely needs a check of helptext

### CODE REVIEW

File: src/app/helptext/directoryservice/activedirectory.ts [hunk 1]
```diff
@@ -48,6 +48,10 @@ activedirectory_certificate_tooltip: T('Select the certificate of the Active Dir
  <a href="--docurl--/system.html#certificates" target="_blank">Certificates</a>\
  menu.'),
 
+ad_validate_certificates_placeholder: T('Validate Certificates'),
+ad_validate_certificates_tooltip: T('Default value is <samp>True</samp>. Checks server certificates in a TLS session. \
```
Comment (wblock) [inline src/app/helptext/directoryservice/activedirectory.ts]:
```suggestion
ad_validate_certificates_tooltip: T('Check server certificates in a TLS session. \
```

File: src/app/helptext/directoryservice/ldap.ts [hunk 2]
```diff
@@ -95,6 +71,10 @@ ldap_certificate_placeholder : T('Certificate'),
 ldap_certificate_tooltip: T('Select the LDAP CA certificate. The certificate for the\
  LDAP server CA must first be imported using the System/Certificates menu.'),
 
+ldap_validate_certificates_placeholder: T('Validate Certificates'),
+ldap_validate_certificates_tooltip: T('Default value is <samp>True</samp>. Checks server certificates in a TLS session. \
```
Comment (wblock) [inline src/app/helptext/directoryservice/ldap.ts]:
```suggestion
ldap_validate_certificates_tooltip: T('Check server certificates in a TLS session. \
```

File: src/app/helptext/directoryservice/ldap.ts [hunk 3]
```diff
@@ -95,6 +71,10 @@ ldap_certificate_placeholder : T('Certificate'),
...
+ Turn off in cases where LDAPs is used.'),
```
Comment (wblock) [inline src/app/helptext/directoryservice/ldap.ts]:
```suggestion
 Unset when LDAPS is used.'),
```

Comment (anodos325) [reply src/app/helptext/directoryservice/ldap.ts]: We shouldn't advise to unset this. Turning off is not a requirement for LDAPs.

Comment (wblock) [reply src/app/helptext/directoryservice/ldap.ts]: Does that apply to AD also?

File: src/app/helptext/directoryservice/activedirectory.ts [hunk 4]
```diff
@@ -48,6 +48,10 @@ activedirectory_certificate_tooltip: T('Select the certificate of the Active Dir
...
+ Turn off in cases where LDAPs is used.'),
```
Comment (wblock) [inline src/app/helptext/directoryservice/activedirectory.ts]:
```suggestion
```

**Pull request merged**